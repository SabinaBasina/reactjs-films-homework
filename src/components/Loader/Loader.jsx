import React from 'react';
import Loading from '../../images/Loading.gif';
import styles from './Loader.scss';

export default ({ loading = true }) => loading
  && (
    <img
      src={Loading}
      className={styles.loader}
      alt="Loading"
    />
  );
