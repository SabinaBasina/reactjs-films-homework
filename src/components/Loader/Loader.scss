.loader {
  display: block;
  margin: 30px auto 0 auto;
  text-align: center;
}
