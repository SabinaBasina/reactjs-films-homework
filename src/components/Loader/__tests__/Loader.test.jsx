import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Loader from '../Loader';

test('Menu snapsphot', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Loader />);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});
