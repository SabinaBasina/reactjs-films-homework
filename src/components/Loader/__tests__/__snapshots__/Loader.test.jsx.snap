// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu snapsphot 1`] = `
<img
  alt="Loading"
  className="loader"
  src="test-file-stub"
/>
`;
